Assets/LeapMotion/Scripts/LeapServiceProvider.cs

```diff
@@ -142,7 +142,7 @@ public class LeapServiceProvider : LeapProvider {
 
         // TODO: Add baseline & offset when included in API
         // NOTE: Alternative is to use device type since all parameters are invariant
-        info.isEmbedded = devices[0].IsEmbedded;
+        info.isEmbedded = devices[0].Type == Device.DeviceType.TYPE_KEYBOARD || devices[0].Type == Device.DeviceType.TYPE_LAPTOP;
```

Comment:
Note that device type isn't actually implemented. But either way, since the service no longer has a concept mapping to "isEmbedded" we might just want to remove it here.